types/react-clock/index.d.ts

```diff
@@ -0,0 +1,112 @@
+// Type definitions for react-clock 3.0
+// Project: https://github.com/wojtekmaj/react-clock
+// Definitions by: Enmanuel Veras <https://github.com/everas7>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+export default function Clock(props: ClockProps): JSX.Element;
+export type ClockValue = string | Date;
+export interface ClockProps<T = ClockValue> {
+    /**
+     * Class name(s) that will be added along with "react-clock" to the main React-Clock <time> element.
+     */
+    className?: string | string[];
+    /**
+     * Hour hand length, in %.
+     * Default: 50
```

Comment:
```ts
     * Default: 50
```

so it begs here and in subsequent usage to use proper JSDOC:
```ts
     * @default 50
```
it just makes a great difference for tooling.

Reply:
Thanks for that observation. Already applied the fix